fred2/fred.rc

```diff
@@ -70,45 +73,78 @@ END
 // Icon with lowest ID value placed first to ensure application icon
 // remains consistent on all systems.
 IDR_MAINFRAME           ICON                    "res\\FRED.ico"
+
 IDR_FREDTYPE            ICON                    "res\\FREDDoc.ico"
 
+
 /////////////////////////////////////////////////////////////////////////////
 //
 // Bitmap
 //
 
 IDR_MAINFRAME           BITMAP                  "res\\Toolbar.bmp"
+
```

Comment:
It was added by the Visual Studio Resource editor so there isn't much I can do about it.